guard.cpp

```diff
@@ -9,13 +9,32 @@ using namespace openpower::guard;
 
 void guardList(bool displayResolved)
 {
+    bool display = false;
```

Comment:
> -When there is no resolved guard record to display, the header
gets displayed. So in this case displaying message
"No Records to display"

When there is only resolved guard records to display, the header
gets displayed. So in this case displaying a message
"No Records to display"